column-grouped/columnGroupedChart.js

```diff
@@ -11,6 +11,7 @@ export function draw() {
     let markers = false; // eslint-disable-line
     let includeMarker = undefined; // eslint-disable-line
     let interpolation = d3.curveLinear;
+    let numbers = true;
```

Comment:
Comments explaining what these do could be useful, `numbers` isn't a very descriptive variable name (could maybe be referred to as `showNumberLabels` or some such, possibly?).

Reply:
Ahh sorry I only put the comment in the index.js file, added